app/models/food.rb

```diff
@@ -0,0 +1,10 @@
+class Food < ApplicationRecord
+  has_many :ingredients, dependent: :destroy
+  has_many :recipes, through: :ingredients
+  belongs_to :user, foreign_key: 'user_id', optional: true
+
+  # validates :name, presence: true
+  # validates :measuring_unit, presence: true
+  # validates :quantity, presence: true, numericality: { greater_than: 0 }
+  # validates :price, presence: true, numericality: { greater_than: 0 }
```

Comment:
### [OPTIONAL]
It's a good practice to validate a model but in case if you don't want to validate the model then kindly remove the commented code to make the file look clean and professional. 👍